From: Kyle McMartin <kmcmarti@redhat.com>
Date: Mon, 2 Jun 2014 15:11:01 -0400
Subject: [PATCH] perf: install trace-event plugins

perf hardcodes $libdir to be lib for all but x86_64, so kludge around it
until upstream gets their act together.
---
 tools/perf/config/Makefile | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/tools/perf/config/Makefile b/tools/perf/config/Makefile
index 86c21a24da46..bf0fe97bd358 100644
--- a/tools/perf/config/Makefile
+++ b/tools/perf/config/Makefile
@@ -642,8 +642,12 @@ endif
 ifeq ($(IS_X86_64),1)
 lib = lib64
 else
+ifdef MULTILIBDIR
+lib = $(MULTILIBDIR)
+else
 lib = lib
 endif
+endif
 libdir = $(prefix)/$(lib)
 
 # Shell quote (do not use $(call) to accommodate ancient setups);
-- 
1.9.3

